src/index.js

```diff
@@ -71,14 +72,17 @@ function main() {
   if (!clusterId) {
     throw 'Please specify process.env.CLUSTER_ID';
   }
-  log.debug({ razeeApi, clusterId });
+  if (razeeUpdateInterval < 60000) {
+    throw `Please use a value greater than or equal to 60000. RAZEE_UPDATE_INTERVAL_MS=${RAZEE_UPDATE_INTERVAL_MS}`;
```

Comment:
I suggest instead of throwing an error, instead log a warning and set the value to 60000 so the service still works

Reply:
Updated now to default to `60000`